<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="author" content="Kelsey Florek">

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-151344274-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-151344274-1');
    </script>

    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

    <title>2022 Virology Conference</title>

    <link rel="stylesheet" href="/resources/revealjs/dist/reset.css">
    <link rel="stylesheet" href="/resources/revealjs/dist/reveal.css">
    <link rel="stylesheet" href="/resources/revealjs/dist/theme/f_theme.css">
    <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400|Noto:400,400i,600" rel="stylesheet">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">

    <!-- Theme used for syntax highlighted code -->
    <link rel="stylesheet" href="/resources/revealjs/plugin/highlight/monokai.css">
  </head>
  <body>
    <div class="reveal">
      <div class="slides">
        <section>
          <h2>Viewing the SARS-CoV-2 pandemic through a genomic lens</h2><br>
          <p>
            Kelsey Florek, PhD, MPH</br>
            Senior Genomics and Data Scientist</br>
            Wisconsin State Laboratory of Hygiene</br>
            June 27, 2022</br></br>
          </p>

          <div style="position:absolute;width: 100%;text-align:center;">
            Slides live at:<br> www.k-florek.net/talks
          </br>
          </br>
          <i class="fab fa-twitter"></i><a href='https://twitter.com/kelsey_florek'>@kelsey_florek</a>
          </div>
          </br>

          <div style="float:left;width:50%">
            <img style="width: 40%;display:block;float:left;margin-top:30px" src="/resources/restalks/images/wslh_logo_white.png">
          </div>
        </section>

        <section data-background-iframe="/resources/restalks/gradient.html">
          <h2 style="color: white">Classification of an evolving virus</h2>
        </section>

        <section>
          <img style="width:70%" src="images/sars-cov-2_evolution.png">
          <div class="citation">
            <a href="https://nextstrain.org/">https://nextstrain.org/</a>
          </div>
        </section>

        <section>
          <h3>Key components of dynamic viral classification</h3>
          <ul>
            <li>capture local and global patterns of genetic diversity in a timely and coherent manner</li>
            <li>track emerging lineages and their movement</li>
            <li>robust and flexible to changes in viral diversity</li>
            <li>ability to incorporate the emergence and disappearance of lineages over time</li>
          </ul>

          <div class="citation">
            <a href="https://doi.org/10.1038/s41564-020-0770-5">Rambaut et al. (2020) <i>Nature Microbiology</i></a>
          </div>
        </section>

        <section>
          <h3>Nextclade</h3>
          <div style="float:left;width:50%">
            <ul>
              <li>A VOC or VOI is recognized by the WHO and given a Greek letter label</li>
              <li>A clade reaches >20% global frequency for 2 or more months</li>
              <li>A clade reaches >30% regional frequency for 2 or more months</li>
              <li>A clade shows consistent >0.05 per day growth in frequency where it is circulating and has reached >5% regional frequency</li>
            </ul>
          </div>
          <div style="float:right;width:50%">
            <img style="width:100%" src="images/ncov_clades_schematic_2022_04_29.png">
          </div>
          <div class="citation">
            <a href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</a>
          </div>
        </section>

        <section>
          <h3>Pangolin</h3>
          <div style="float:left;width:50%">
            <ul>
              <li>Pango Committee maintains the "rules" of the nomenclature system</li>
              <li>Pango lineages are designed to represent the branches of the pandemic tree</li>
              <li>Pango lineages are intended to highlight epidemiologically-relevant events</li>
              <li>Pango lineages can be classified through two major methods
                <ul>
                  <li>pangoLEARN: random forest based decision model</li>
                  <li>UShER: sample placement on pre-existing phylogenetic trees</li>
                </ul>
              </li>
            </ul>
          </div>
          <div style="float:right;width:50%">
            <img style="width:100%" src="images/pango_designations.png">
          </div>
          <div class="citation">
            <a href="https://www.pango.network/">https://www.pango.network/</a>
          </div>
        </section>

        <section>
          <h3>WHO Naming Convention</h3>
          
          <div style="float:left;width:50%">
            <div><h4>Currently circulating variants of concern (VOCs):</h4><div style="overflow-x:auto;font-size:small"><table style="border-width:1px;border-style:solid;border-color:#080808;width:100%;"><thead><tr style="height:68.8333px;"><th data-role="resizable" style="width:94.312px;border-width:0px;border-style:solid;border-color:#121212;background-color:#00bbea;text-align:left;vertical-align:top;"><p style="text-align:left;">WHO&nbsp;label<strong>&nbsp;</strong><strong></strong></p></th><th style="width:10.3444%;border-width:0px;border-style:solid;border-color:#151515;background-color:#1cbde5;text-align:left;" data-role="resizable">Pango&nbsp;<strong>&nbsp;</strong><strong></strong><br /><strong>lineage</strong>&bull;<strong></strong></th><th style="width:12.2127%;border-width:0px;border-style:solid;border-color:#0d0c0c;background-color:#1cbde5;text-align:left;" data-role="resizable">GISAID clade<strong></strong></th><th style="width:14.4777%;border-width:0px;border-style:solid;border-color:#080808;background-color:#1cbde5;text-align:left;" data-role="resizable">Nextstrain&nbsp;<strong>clade</strong><strong>&nbsp;</strong><strong></strong></th><th style="width:19.4988%;border-width:0px;border-style:solid;border-color:#080808;background-color:#1cbde5;text-align:left;" data-role="resizable"><strong>Additional&nbsp;</strong><strong>amino acid changes&nbsp;</strong><strong>monitored</strong><strong>&deg;</strong></th><th style="width:12.5853%;border-width:0px;border-style:solid;border-color:#080808;background-color:#1cbde5;text-align:left;" data-role="resizable">Earliest documented&nbsp;<strong>&nbsp;</strong><strong></strong><br /><strong>samples</strong><strong>&nbsp;</strong><strong></strong></th><th style="width:19.6989%;border-width:0px;border-style:solid;border-color:#0d0c0c;background-color:#1cbde5;text-align:left;">Date of designation<strong>&nbsp;</strong><strong></strong></th></tr></thead><tbody><tr style="height:164.333px;"><td style="width:11.1877%;text-align:left;">Omicron*</td><td style="width:10.3444%;text-align:left;vertical-align:middle;">B.1.1.529</td><td style="width:12.2127%;text-align:left;">GR/484A</td><td style="width:14.4777%;text-align:left;"><p>&nbsp;</p>21K, 21L, 21M,&nbsp;, 22A, 22B, 22C<p>&nbsp;</p></td><td style="width:19.4988%;text-align:left;"><p>+S:R346K</p><p>+S:L452X</p><p>+S:F486V</p></td><td style="width:12.5853%;text-align:left;">Multiple countries, Nov-2021</td><td style="width:19.6989%;text-align:left;"><p>VUM: 24-Nov-2021</p><p>VOC: 26-Nov-2021</p></td></tr></tbody></table></div>
            <p style="font-size:small">* Includes BA.1, BA.2, BA.3, BA.4, BA.5 and&nbsp;descendent lineages. It also includes BA.1/BA.2 circulating recombinant forms such as XE. WHO emphasizes that these descendant lineages should be monitored as distinct lineages by public health authorities and comparative assessments of their virus characteristics should be undertaken.</p>
            <p style="font-size:small">The full list of Pango lineages can be found here:&nbsp;<a href="https://cov-lineages.org/lineage_list.html" data-sf-ec-immutable="">https://cov-lineages.org/lineage_list.html</a>; for FAQ, visit: <a href="https://www.pango.network/faqs/" data-sf-ec-immutable="">https://www.pango.network/faqs/</a></p>
            <p style="font-size:small">&deg; Only found in a subset of sequences<sup></sup></p></div>
          </div>

          <div class="citation" style="left:0px;right:auto;bottom:75px">
            <a href="https://www.who.int/activities/tracking-SARS-CoV-2-variants">https://www.who.int/activities/tracking-SARS-CoV-2-variants</a>
          </div>

          <div style="float:right;width:49%">
            <div><h4>Previously circulating VOCs:</h4><div style="overflow-x:auto;font-size:small"><table style="border-width:1px;border-style:solid;border-color:#080808;width:100%;"><thead><tr style="height:68.8333px;"><th data-role="resizable" style="width:94.312px;border-width:0px;border-style:solid;border-color:#121212;background-color:#00bbea;text-align:left;vertical-align:top;"><p style="text-align:left;">WHO&nbsp;label<strong>&nbsp;</strong><strong></strong></p></th><th style="width:10.3444%;border-width:0px;border-style:solid;border-color:#151515;background-color:#1cbde5;text-align:left;" data-role="resizable">Pango&nbsp;<strong>&nbsp;</strong><strong></strong><br /><strong>lineage</strong>&bull;<strong></strong></th><th style="width:12.2127%;border-width:0px;border-style:solid;border-color:#0d0c0c;background-color:#1cbde5;text-align:left;" data-role="resizable">GISAID clade<strong></strong></th><th style="width:14.4777%;border-width:0px;border-style:solid;border-color:#080808;background-color:#1cbde5;text-align:left;" data-role="resizable">Nextstrain&nbsp;<strong>clade</strong><strong>&nbsp;</strong><strong></strong></th><th style="width:12.5853%;border-width:0px;border-style:solid;border-color:#080808;background-color:#1cbde5;text-align:left;" data-role="resizable">Earliest documented&nbsp;<strong>&nbsp;</strong><strong></strong><br /><strong>samples</strong><strong>&nbsp;</strong><strong></strong></th><th style="width:19.6989%;border-width:0px;border-style:solid;border-color:#0d0c0c;background-color:#1cbde5;text-align:left;">Date of designation<strong>&nbsp;</strong><strong></strong></th></tr></thead><tbody><tr style="height:128.542px;"><td style="width:94.312px;text-align:left;vertical-align:middle;"><p>Alpha&nbsp;<br /></p></td><td style="width:10.3444%;text-align:left;vertical-align:middle;"><p>B.1.1.7&nbsp;</p></td><td style="width:12.2127%;text-align:left;vertical-align:middle;"><p>GRY </p></td><td style="width:14.4777%;text-align:left;vertical-align:middle;"><p>20I (V1)&nbsp; </p></td><td style="width:12.5853%;text-align:left;vertical-align:top;"><p>United Kingdom,&nbsp;&nbsp;<br />Sep-2020&nbsp; </p></td><td style="width:166.062px;text-align:left;vertical-align:middle;"><p>VOC: 18-Dec-2020</p><p>Previous VOC: 09-Mar-2022</p></td></tr><tr style="height:109.542px;"><td style="width:11.1877%;background-color:#69d4ef;text-align:left;vertical-align:middle;">Beta&nbsp;<br /></td><td style="width:10.3444%;background-color:#69d4ef;text-align:left;"><p>B.1.351&nbsp;<br /></p></td><td style="width:12.2127%;background-color:#69d4ef;text-align:left;">GH/501Y.V2&nbsp; </td><td style="width:14.4777%;background-color:#69d4ef;text-align:left;">20H (V2)</td><td style="width:12.5853%;background-color:#69d4ef;text-align:left;vertical-align:middle;">South Africa,&nbsp;&nbsp;<br />May-2020&nbsp; </td><td style="width:19.6989%;background-color:#69d4ef;text-align:left;"><p>VOC: 18-Dec-2020</p><p>Previous VOC: 09-Mar-2022</p></td></tr><tr style="height:128.542px;"><td style="width:11.1877%;text-align:left;">Gamma&nbsp;<br /></td><td style="width:10.3444%;text-align:left;"><p>P.1&nbsp;<br /></p></td><td style="width:12.2127%;text-align:left;">GR/501Y.V3&nbsp; </td><td style="width:14.4777%;text-align:left;">20J (V3) </td><td style="width:12.5853%;text-align:left;">Brazil,&nbsp;&nbsp;<br />Nov-2020&nbsp; </td><td style="width:19.6989%;text-align:left;"><p>VOC: 11-Jan-2021</p><p>Previous VOC: 09-Mar-2022</p></td></tr><tr style="height:52.9167px;"><td style="width:11%;text-align:left;vertical-align:middle;background-color:#69d4ef;">Delta&nbsp;</td><td style="width:10%;text-align:left;vertical-align:middle;background-color:#69d4ef;">B.1.617.2</td><td style="width:12%;text-align:left;vertical-align:middle;background-color:#69d4ef;">G/478K.V1</td><td style="width:14%;text-align:left;vertical-align:middle;background-color:#69d4ef;">21A, 21I, 21J</td><td style="width:12%;text-align:left;vertical-align:middle;background-color:#69d4ef;">India,&nbsp;&nbsp;<br />Oct-2020&nbsp;</td><td style="width:19%;text-align:left;vertical-align:middle;background-color:#69d4ef;"><p>VOI: 4-Apr-2021&nbsp;<br />VOC: 11-May-2021</p><p>Previous VOC: 7-Jun-2022</p></td></tr></tbody></table></div>
            <p style="font-size: small;">*Includes all descendent lineages. <br /></p></div>
          </div>

          
        </section>

        <section data-background-iframe="/resources/restalks/gradient.html">
          <h2 style="color: white">Current SARS-CoV-2 Situation Report</h2>
        </section>

        <section>
          <h3>Wisconsin SARS-CoV-2 Genomic Dashboard</h3>
          <img style="width:100%" src="images/sc2_dashboard.png">
          <div class="citation">
            <a href="https://dataportal.slh.wisc.edu/sc2dashboard">https://dataportal.slh.wisc.edu/sc2dashboard</a>
          </div>
        </section>

        <section>
          <h3>Wisconsin Lineages</h3>
          <img style="width:100%" src="images/variants.png">
          <div class="citation">
            <a href="https://dataportal.slh.wisc.edu/sc2dashboard">https://dataportal.slh.wisc.edu/sc2dashboard</a>
          </div>
        </section>

        <section data-transition="slide-in fade-out">
          <h3>Emergence of Omicron</h3>
          <div style="float:left;width:50%">
            <img style="width:100%" src="images/sc2_rootless_20220621.png">
          </div>
          <div style="float:right;width:50%" class="fragment">
            <br>
            <br>
            <br>
            <p><b>Three Theories</b></p>
            <ul>
              <li>Undetected Spread</li>
              <li>Animal Reservoir (Mouse/Rat)</li>
              <li>Chronic Infection</li>
            </ul>
          </div>
          <div class="citation">
            <a href="https://doi.org/10.1038/d41586-022-00215-2">https://doi.org/10.1038/d41586-022-00215-2</a>
            <br>
            <a href="https://nextstrain.org/">https://nextstrain.org/</a>
          </div>
        </section>

        <section data-transition="fade-in fade-out">
          <h3>Omicron Today</h3>
          <div style="float:left;width:50%">
            <img style="width:100%" src="images/sc2_rootless_20220621.png">
          </div>
          <div class="citation">
            <a href="https://nextstrain.org/">https://nextstrain.org/</a>
          </div>
        </section>

        <section data-transition="fade-in fade-out">
          <h3>Omicron Today</h3>
          <div style="float:left;width:50%">
            <img style="width:100%" src="images/sc2_rootless_20220621_target.png">
          </div>
          <div style="float:left;width:50%">
            <img style="width:100%" src="images/sc2_omicron.png">
          </div>
          <div class="citation">
            <a href="https://nextstrain.org/">https://nextstrain.org/</a>
          </div>
        </section>

        <section>
          <h3>Omicron Today</h3>
          <div style="float:left;width:70%">
            <img style="width:100%" src="images/omicron_sublineages.png">
          </div>
          <div style="float:right;width:30%">
            <img style="width:100%" src="images/omicron_nature.png">
            <p style="font-size:medium">"Importantly, BA.2.12.1 and BA.4/BA.5 display stronger neutralization evasion than BA.2 against the plasma from 3-dose vaccination and, most strikingly, from post-vaccination BA.1 infections."</p>
          </div>

          <div class="citation">
            <a href="https://dataportal.slh.wisc.edu/sc2dashboard">https://dataportal.slh.wisc.edu/sc2dashboard</a>
            <br>
            <a href="https://doi.org/10.1038/s41586-022-04980-y">Cao et al. 2022 <i>Nature</i></a>
          </div>
        </section>

        <section data-background="#0969da" style="color:white">
          <h3>Summary</h3>
          <ul>
            <li class="fragment">The adaptive landscape for SARS-CoV-2 has been difficult to predict or measure.</li>
            <li class="fragment">It is very likely we will see new immune escape variants continue to emerge.</li>
            <li class="fragment">Increases in at home testing has the potential to bias genomic surveillance.</li>
          </ul>
        </section>

        <section data-background-iframe="/resources/restalks/gradient.html">
          <h2 style="color: white">Challenges in the application of genomic surveillance data</h2>
        </section>

        <section>
          <h3>Public Health is currently a series of Data Silos</h3>
          <image style="width:80%" src="images/data_silos.png">
        </section>

        <section>
          <h3>Integrating environmental, epidemiological, and genomic data with a data lake</h3>
          <div style="float:left;width:50%">
            <p><b>data lake:</b> centralized repository designed to store, process, and secure large amounts of structured and unstructured data at any scale</p>
            <ul class="fragment">
              <li>Data Lakes retain all data</li>
              <li>Support all data types</li>
              <li>Adapt easily to changes</li>
              <li>Require data scientists to navigate</li>
              <li>Require proper management to prevent a data swamp</li>
            </ul>
          </div>

          <div style="float:right;width:50%">
            <image style="width:80%" src="/talks/2022-05-16-arln-genomic-reporting/images/Genomic_Data_Lake.png">
          </div>
        </section>

        <section>
          <h3>Integrated data better supports public health</h3>
          <ul>
            <li>Identify correlations between genetic variants and vaccination breakthroughs or increased symptom severity.</li>
            <li>Faster and more informed outbreak response.</li>
            <li>Application of machine learning models and AI to identify genomic patterns.</li>
            <li>Big data enables the application of precision public health efforts.</li>
          </ul>
        </section>

        <section>
          <h3>Acknowledgments</h3>
          <div style="width:50%;float:left;">
            <div class="aligned" style="margin-left: 80px">
              <image style="width:140px;border-radius:50%;box-shadow:2px 2px 5px black" src="/images/site/as_photo.jpg">
              <span style="margin-left:30px;text-align:left">Abigail Shockey, PhD</br></span>
            </div>
            <div class="aligned" style="margin-left: 80px">
              <image style="width:140px;border-radius:50%;box-shadow:2px 2px 5px black" src="/images/site/cj_photo.jpg">
              <span style="margin-left:30px;text-align:left">Christopher Jossart, MPH</br></span>
            </div>
          </div>
          <div style="width:50%;float:right;">
            </br>
            </br>
            <ul>
              <li>WSLH Virus Sequencing Group
                <ul>
                  <li>Tim Davis</li>
                  <li>Rich Griesser</li>
                  <li>Kyley Guenther</li>
                  <li>Erika Hanson</li>
                </ul>
              </li>
              <li>Special Thanks
                <ul>
                  <li>Erika Hanson</li>
                </ul>
              </li>
            </ul>
          </div>

          <div class="citation">
            <a href="https://revealjs.com/">Reveal.JS</a>
          </div>
        </section>

        <section data-background-iframe="/resources/restalks/paperjs/viruses.html">
          <h2 style="color: black">Questions?</h2>
        </section>

      </div>
    </div>

    <script src="/resources/revealjs/dist/reveal.js"></script>
    <script src="/resources/revealjs/plugin/notes/notes.js"></script>
    <script src="/resources/revealjs/plugin/markdown/markdown.js"></script>
    <script src="/resources/revealjs/plugin/highlight/highlight.js"></script>
    <script>
      // More info about initialization & config:
      // - https://revealjs.com/initialization/
      // - https://revealjs.com/config/
      Reveal.initialize({
        hash: true,
        history: false,
        controls: true,
        center: true,
        progress: true,
        slideNumber: true,
        touch: true,
        viewDistance: 6,
        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1280,
        height: 720,

        // Factor of the display size that should remain empty around the content
        margin: 0.05,

        // Bounds for smallest/largest possible scale to apply to content
        minScale: 0.1,
        maxScale: 5,

        // Learn about plugins: https://revealjs.com/plugins/
        plugins: [ RevealMarkdown, RevealHighlight, RevealNotes ]
      });
    </script>
  </body>
</html>
